src/components/DriverCard.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+function DriverCard(props) {
+    const {name, rating, img, car} = props;
+  return (
+    <div>
+            <h1>{name}</h1>
+            <h2>{rating}</h2>
```

Comment:
Here, you could've also called the "Rating.jsx" component to render stars instead of plain numbers